packages/mui-joy/src/ListSubheader/ListSubheaderContext.ts

```diff
@@ -4,4 +4,8 @@ const ListSubheaderDispatch = React.createContext<
   undefined | React.Dispatch<React.SetStateAction<string>>
 >(undefined);
 
+if (process.env.NODE_ENV !== 'production') {
+  ListSubheaderDispatch.displayName = 'ListSubheaderDispatch';
```

Comment:
There's an inconsistency between file name and the export. Since it's a default export, I'd rather use the name equal to file name. CC @siriwatknp

Reply:
Ah correct 
```suggestion
  ListSubheaderContext.displayName = 'ListSubheaderContext';
```

Reply:
Updated to follow the convention, and the file name, it should be `Context` as a suffix.